### ISSUE TITLE
Title for the facetted search

### ISSUE THREAD

**Issue by pilz** [ploneintranet/ploneintranet.prototype#140]
During first user contact we saw that users didn't notice the facets. Would it make sense to add a heading like "Narrow your search" above them?

**Comment by angeldasangel** [comment]
@cornae 
Would that be an easy addition?

**Comment by cornae** [comment]
Yes it's an easy addition. But if users don't notice an entire column full of controls, I doubt if a header would help much.
